src/auth/authController.ts

```diff
@@ -13,3 +14,51 @@ export const refreshAccessToken = async (req: Request, res: Response) => {
   const tokens = await authService.refreshAccessToken(refreshToken);
   return res.json(tokens);
 };
+
+/**
+ * @desc Links a GET/CBORD account to a deviceId via a PIN.
+ * This is the one-time setup for the finance feature.
+ */
+export const getAuthorize = async (
+  req: Request,
+  res: Response,
+  next: NextFunction,
```

Comment:
See below comment. You can simplify the method signature to only accept `req` and `res` (eliminating the `next` parameters - also make sure to remove the import for it)